sdk/containerregistry/Azure.Containers.ContainerRegistry/api/Azure.Containers.ContainerRegistry.netstandard2.0.cs

```diff
@@ -300,6 +320,12 @@ public partial class OciBlobDescriptor
         public string MediaType { get { throw null; } set { } }
         public long? Size { get { throw null; } set { } }
     }
+    public partial class OciIndex : Azure.Containers.ContainerRegistry.Specialized.ArtifactManifest
```

Comment:
I will come up with a swagger transform that removes the ArtifactManifest type and makes it so there is no inheritance for OciIndex or OciManifest.

Reply:
This is implemented here: https://github.com/Azure/azure-sdk-for-net/pull/31908/files#diff-0ca3c36de4ce63c593836ed47c3d657a7cb2ad38791277414b440ea7a6cb5007L101